src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioDiagnosticListTable.cs

```diff
@@ -144,13 +144,17 @@ private void ConnectToBuildUpdateSource(ExternalErrorDiagnosticUpdateSource erro
 
                 SetStableState(errorSource.IsInProgress);
 
-                errorSource.DiagnosticsUpdated += OnDiagnosticsUpdated;
                 errorSource.BuildStarted += OnBuildStarted;
             }
 
             private void OnBuildStarted(object sender, bool started)
```

Comment:
@mavasani started == true means build started, started == false means build finished. we can change name but I will do that in different checkin in master